third_party/static/messageformat-0.3.1/doc/MessageFormat.formatters.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
```

Comment:
All the third-party stuff should not be part of the PR. Look at manifest.json for how to include new libs.